spec/intro.dd

```diff
@@ -10,59 +10,47 @@ to create native libraries or executables.)
 $(P This document is the reference manual for the D Programming Language. For more information and
 other documents$(COMMA) see $(LINK2 https://dlang.org/, The D Language Website).)
 
-$(H2 Phases of Compilation)
-
-$(P The process of compiling is divided into multiple phases. Each phase has no
-dependence on subsequent phases. For example, the scanner is not perturbed by
-the semantic analyzer. This separation of the passes makes language tools like
-syntax directed editors relatively easy to produce. It also is possible to
-compress D source by storing it in $(SINGLEQUOTE tokenized) form.)
+$(H2 Program Structure)
 
-$(OL
-        $(LI $(B source character set)$(BR)
```

Comment:
Why delete the entries on source character set and script line?

Reply:
Hi @atilaneves , I explained this in the previous pull request, here it is again. Basically to pick a couple of random details from a particular phase seems pointless as this is covered in more detail in the lexical analysis section. The description in the introduction ought to be a high level overview. These two points seemed very specific to me and do not add to the general overview of the process.